### DISCUSSION TITLE
problem fetching Qgis data

### DISCUSSION THREAD

**Discussion by AlvBuoro** [microsoft/PlanetaryComputer#159] [Q&A]
under sTac Api browser the search of Microsoft planetary computer Stac Api gives the error details, None.
I installed the updated version of the plugin, could this be the cause?

**Comment by TomAugspurger** [comment]
> Microsoft planetary computer Stac Api gives the error details

Is that the exact text from the error? Can you share a screenshot of what you're seeing?

**Reply by AlvBuoro** [reply]
<img width="576" alt="error" src="https://user-images.githubusercontent.com/111052001/211592096-a864739d-f48e-4f00-a5b1-36a8e3d43130.png">

**Reply by TomAugspurger** [reply]
Interesting. I just upgraded the plugin and can confirm it's working for me:

<img width="405" alt="image" src="https://user-images.githubusercontent.com/1312546/211617081-6d538fca-672f-43be-b2a4-4cb938e8d50c.png">

A few suggestions:

1. Make sure you can access the Planetary Computer's STAC API by visiting http://planetarycomputer.microsoft.com/api/stac/v1/ in your browser.
2. See if you can browse other STAC APIs than the Planetary Computer's
3. Follow the [help instructions](https://stac-utils.github.io/qgis-stac-plu[gin/#help) from the developers of that plugin.

**Reply by AlvBuoro** [reply]
Today I restarted another session and everything is working. I have not changed anything in particular since yesterday

**Comment by AlvBuoro** [comment]
HI Tom,

Thanks for the concern, here it is the message:
***@***.***
It appears for any item in the collection I select as well as for other connections other than Microsoft Planetary
Regards
Alvaro B Buoro
Pronouns:<https://www.mottmac.com/article/66545/person-pronouns> he, him, his
Climate Resilience Data Specialist
D +441223463942

**Comment by mdesalew** [comment]
Hello,

I am having the same issue with the STAC API Browser. My Microsoft Planetary Computer STAC API gives this error:  
"Problem in fetching content for https://planetarycomputer.microsoft.com/api/stac/v1.Error details, 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte"

Thank you for any help!

**Reply by TomAugspurger** [reply]
I'm not familiar with how that plugin is implemented, but I don't think we've changed anything recently that would affect that. Might want to check on https://github.com/stac-utils/qgis-stac-plugin/.
